### DISCUSSION TITLE
2FA to be required on Git LFS organization

### DISCUSSION THREAD

**Discussion by bk2204** [git-lfs/git-lfs#5312] [Announcements]
The Core Team (@git-lfs/core) is going to be requiring 2FA for all organization members as of Friday, March 24, 2023.  Mostly, this affects members of @git-lfs/implementers, since everyone not on that team already has 2FA enabled.  Anyone who doesn't have 2FA enabled by that point will be removed from the organization.  As you might have guessed, this is for security reasons.

Note that this doesn't affect regular users reporting issues, making pull requests, fetching, cloning, or otherwise working with public resources, beyond what GitHub will already be requiring in the future.  Only people on a team in the `git-lfs` org will be affected.

**Comment by bturner** [comment]
My 2FA is all in order, but can you remove me from those LFS teams I'm a member of? I no longer work for Atlassian.

**Reply by bk2204** [reply]
Done.
